haystack/components/eval/statistical_evaluator.py

```diff
@@ -9,6 +9,19 @@
 from haystack.core.component import component
 
 
+class StatisticalMetric(Enum):
+    """
+    Metrics supported by the StatisticalEvaluator.
+    """
+
+    F1 = "f1"
+    EM = "exact_match"
+
+    @classmethod
+    def from_string(cls, metric: str) -> "StatisticalMetric":
```

Comment:
```suggestion
    def from_str(cls, metric: str) -> "StatisticalMetric":
```
We use this convention in other places.